src/makim/console.py

```diff
@@ -2,8 +2,12 @@
 
 import os
 
+from typing import Tuple
 
-def get_terminal_size(default_size=(80, 24)):
+
+def get_terminal_size(
+    default_size: Tuple[int, int] = (80, 24),
+) -> Tuple[int, int]:
```

Comment:
this could be `tuple` if you import `from __future__ import annotations`
but I will not block this PR to be merged because of this

Reply:
actually there are two more changes for this PR before it is ready to go .. so please could you already fix this one as well?